tasks.py

```diff
@@ -235,7 +229,9 @@ def _should_ignore(path):
         if not _should_ignore(path)
     ]
 
-    run(["kubeval"] + kube_yamls)
+    skip_kinds = ",".join(devops.settings.KUBEVAL_SKIP_KINDS)
```

Comment:
I imagine this does not work well if `devops.settings.KUBEVAL_SKIP_KINDS = []`

Reply:
If you configure it like that, then `skip_kinds` will be an empty string. So the end result is more or less equal to running something like `kubeval --skip-kinds "" kube/02-kured.yaml` which works nicely. This is actually one of the benefits of the subprocess.run, that it takes care of quoting attributes properly. For more details please see https://docs.python.org/3/library/subprocess.html#frequently-used-arguments